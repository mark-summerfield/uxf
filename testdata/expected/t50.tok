FileComment Str("File-level comment")
ListBegin
ListEnd
Eof
----------------------------------------
