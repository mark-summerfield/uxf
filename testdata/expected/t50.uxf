uxf 1
#<File-level comment>
[]
