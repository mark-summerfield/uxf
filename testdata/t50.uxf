uxf 1.0
#<File-level comment>
[]
