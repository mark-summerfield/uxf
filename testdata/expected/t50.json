{
  "UXF^comment": "File-level comment",
  "UXF^data": []
}